src/graph.rs

```diff
@@ -1254,19 +1259,19 @@ impl PyGraph {
 
     /// Generate a dot file from the graph
     ///
-    /// :param node_attr: A callable that will take in a node data object
+    /// :param Callable node_attr: A callable that will take in a node data object
```

Comment:
For this specific one, leave as is as the Callable is known to be kind of a pain. We have a more specific definition in the type stubs